packages/react-composites/src/composites/CallComposite/adapter/CallAdapter.ts

```diff
@@ -247,6 +252,23 @@ export interface VideoBackgroundImage {
   tooltipText?: string;
 }
 
+/* @conditional-compile-remove(video-background-effects) */
+/**
+ * Contains the attibutes of a selected video background effect
+ *
+ * @beta
+ */
+export interface VideoBackgroundEffect {
+  /**
+   * Type of video background effect, can be Blur, Replacement or None.
+   */
+  type: 'Blur' | 'Replacement' | 'None';
```

Comment:
is replacement to represent the backgrounds that contoso would upload as well? they are able to provide their own images right? (not custom user uploads)